[2024-01-11 16:10:03,092] [INFO] [test_01_workspace_mnt.py] [line 44] 插件管理-预览系统插件图标请求路径为:/OneHit/plugin/二维码-生成器/icon/preview/二维码生成器.svg?isSystemInternal=false,请求体为:{}
--- Logging error ---
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 70, in emit
    self.doRollover()
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 171, in doRollover
    self.rotate(self.baseFilename, dfn)
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 111, in rotate
    os.rename(source, dest)
PermissionError: [WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'D:\\D\\api_auto_test\\api_auto_test\\logs\\2024-01-11.log' -> 'D:\\D\\api_auto_test\\api_auto_test\\logs\\2024-01-11.log.1'
Call stack:
  File "D:/D/api_auto_test/api_auto_test/run.py", line 30, in <module>
    run_case()
  File "D:/D/api_auto_test/api_auto_test/run.py", line 22, in run_case
    pytest.main(['--alluredir', 'reports/allure-report'])
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\config\__init__.py", line 169, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 318, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 271, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 325, in _main
    config.hook.pytest_runtestloop(session=session)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 350, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 114, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 133, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 222, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 261, in call_runtest_hook
    return CallInfo.from_call(
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 341, in from_call
    result: Optional[TResult] = func()
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 262, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 169, in pytest_runtest_call
    item.runtest()
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\python.py", line 1792, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\python.py", line 194, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File "D:\D\api_auto_test\api_auto_test\tests\test_01_workspace_mnt.py", line 44, in test_01_workspace_mnt
    logger.info("{}请求路径为:{},请求体为:{}".format(datas["title"], datas["path"], datas["data"]))
Message: '插件管理-预览系统插件图标请求路径为:/OneHit/plugin/二维码-生成器/icon/preview/二维码生成器.svg?isSystemInternal=false,请求体为:{}'
Arguments: ()
[2024-01-11 16:10:03,119] [INFO] [test_01_workspace_mnt.py] [line 77] 插件管理-预览系统插件图标状态码：200，响应结果:<?xml version="1.0" encoding="UTF-8"?>
<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <title>切片</title>
    <g id="-----插件" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="插件图标" transform="translate(-393.000000, -702.000000)">
            <g id="编组" transform="translate(393.000000, 702.000000)">
                <rect id="矩形" x="0" y="0" width="16" height="16"></rect>
                <path d="M5.66666667,9.08333333 C6.01184464,9.08333333 6.32434464,9.22324435 6.55055014,9.44944986 C6.77675565,9.67565536 6.91666667,9.98815536 6.91666667,10.3333333 L6.91666667,13.25 C6.91666667,13.595178 6.77675565,13.907678 6.55055014,14.1338835 C6.32434464,14.360089 6.01184464,14.5 5.66666667,14.5 L2.75,14.5 C2.40482203,14.5 2.09232203,14.360089 1.86611652,14.1338835 C1.63991102,13.907678 1.5,13.595178 1.5,13.25 L1.5,10.3333333 C1.5,9.98815536 1.63991102,9.67565536 1.86611652,9.44944986 C2.09232203,9.22324435 2.40482203,9.08333333 2.75,9.08333333 Z M5.66666667,1.5 C6.01184464,1.5 6.32434464,1.63991102 6.55055014,1.86611652 C6.77675565,2.09232203 6.91666667,2.40482203 6.91666667,2.75 L6.91666667,5.66666667 C6.91666667,6.01184464 6.77675565,6.32434464 6.55055014,6.55055014 C6.32434464,6.77675565 6.01184464,6.91666667 5.66666667,6.91666667 L2.75,6.91666667 C2.40482203,6.91666667 2.09232203,6.77675565 1.86611652,6.55055014 C1.63991102,6.32434464 1.5,6.01184464 1.5,5.66666667 L1.5,2.75 C1.5,2.40482203 1.63991102,2.09232203 1.86611652,1.86611652 C2.09232203,1.63991102 2.40482203,1.5 2.75,1.5 Z M13.25,1.5 C13.595178,1.5 13.907678,1.63991102 14.1338835,1.86611652 C14.360089,2.09232203 14.5,2.40482203 14.5,2.75 L14.5,5.66666667 C14.5,6.01184464 14.360089,6.32434464 14.1338835,6.55055014 C13.907678,6.77675565 13.595178,6.91666667 13.25,6.91666667 L10.3333333,6.91666667 C9.98815536,6.91666667 9.67565536,6.77675565 9.44944986,6.55055014 C9.22324435,6.32434464 9.08333333,6.01184464 9.08333333,5.66666667 L9.08333333,2.75 C9.08333333,2.40482203 9.22324435,2.09232203 9.44944986,1.86611652 C9.67565536,1.63991102 9.98815536,1.5 10.3333333,1.5 Z" id="形状结合" stroke="#0854A1" fill-rule="nonzero"></path>
            </g>
        </g>
    </g>
</svg>
--- Logging error ---
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 70, in emit
    self.doRollover()
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 171, in doRollover
    self.rotate(self.baseFilename, dfn)
  File "C:\Program Files\Python38\lib\logging\handlers.py", line 111, in rotate
    os.rename(source, dest)
PermissionError: [WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'D:\\D\\api_auto_test\\api_auto_test\\logs\\2024-01-11.log' -> 'D:\\D\\api_auto_test\\api_auto_test\\logs\\2024-01-11.log.1'
Call stack:
  File "D:/D/api_auto_test/api_auto_test/run.py", line 30, in <module>
    run_case()
  File "D:/D/api_auto_test/api_auto_test/run.py", line 22, in run_case
    pytest.main(['--alluredir', 'reports/allure-report'])
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\config\__init__.py", line 169, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 318, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 271, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 325, in _main
    config.hook.pytest_runtestloop(session=session)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\main.py", line 350, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 114, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 133, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 222, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 261, in call_runtest_hook
    return CallInfo.from_call(
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 341, in from_call
    result: Optional[TResult] = func()
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 262, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\runner.py", line 169, in pytest_runtest_call
    item.runtest()
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\python.py", line 1792, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\pluggy\_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\wangsihan\AppData\Roaming\Python\Python38\site-packages\_pytest\python.py", line 194, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File "D:\D\api_auto_test\api_auto_test\tests\test_01_workspace_mnt.py", line 77, in test_01_workspace_mnt
    logger.info("{}状态码：{}，响应结果:{}".format(datas["title"], resp.status_code, resp.text))
Message: '插件管理-预览系统插件图标状态码：200，响应结果:<?xml version="1.0" encoding="UTF-8"?>\n<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">\n    <title>切片</title>\n    <g id="-----插件" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">\n        <g id="插件图标" transform="translate(-393.000000, -702.000000)">\n            <g id="编组" transform="translate(393.000000, 702.000000)">\n                <rect id="矩形" x="0" y="0" width="16" height="16"></rect>\n                <path d="M5.66666667,9.08333333 C6.01184464,9.08333333 6.32434464,9.22324435 6.55055014,9.44944986 C6.77675565,9.67565536 6.91666667,9.98815536 6.91666667,10.3333333 L6.91666667,13.25 C6.91666667,13.595178 6.77675565,13.907678 6.55055014,14.1338835 C6.32434464,14.360089 6.01184464,14.5 5.66666667,14.5 L2.75,14.5 C2.40482203,14.5 2.09232203,14.360089 1.86611652,14.1338835 C1.63991102,13.907678 1.5,13.595178 1.5,13.25 L1.5,10.3333333 C1.5,9.98815536 1.63991102,9.67565536 1.86611652,9.44944986 C2.09232203,9.22324435 2.40482203,9.08333333 2.75,9.08333333 Z M5.66666667,1.5 C6.01184464,1.5 6.32434464,1.63991102 6.55055014,1.86611652 C6.77675565,2.09232203 6.91666667,2.40482203 6.91666667,2.75 L6.91666667,5.66666667 C6.91666667,6.01184464 6.77675565,6.32434464 6.55055014,6.55055014 C6.32434464,6.77675565 6.01184464,6.91666667 5.66666667,6.91666667 L2.75,6.91666667 C2.40482203,6.91666667 2.09232203,6.77675565 1.86611652,6.55055014 C1.63991102,6.32434464 1.5,6.01184464 1.5,5.66666667 L1.5,2.75 C1.5,2.40482203 1.63991102,2.09232203 1.86611652,1.86611652 C2.09232203,1.63991102 2.40482203,1.5 2.75,1.5 Z M13.25,1.5 C13.595178,1.5 13.907678,1.63991102 14.1338835,1.86611652 C14.360089,2.09232203 14.5,2.40482203 14.5,2.75 L14.5,5.66666667 C14.5,6.01184464 14.360089,6.32434464 14.1338835,6.55055014 C13.907678,6.77675565 13.595178,6.91666667 13.25,6.91666667 L10.3333333,6.91666667 C9.98815536,6.91666667 9.67565536,6.77675565 9.44944986,6.55055014 C9.22324435,6.32434464 9.08333333,6.01184464 9.08333333,5.66666667 L9.08333333,2.75 C9.08333333,2.40482203 9.22324435,2.09232203 9.44944986,1.86611652 C9.67565536,1.63991102 9.98815536,1.5 10.3333333,1.5 Z" id="形状结合" stroke="#0854A1" fill-rule="nonzero"></path>\n            </g>\n        </g>\n    </g>\n</svg>'
Arguments: ()
